backend/backend/settings.py

```diff
@@ -148,7 +149,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Europe/Helsinki'
```

Comment:
Probably does not matter here but just a heads up, for the db we usually want the UTC so no matter the timezone we are providing this service to, we can easily convert the UTC to local time.

Reply:
this was done because I was annoyed that the expires_ts wasn't showing the Finnish time, but I can revert it back to how it was earlier.

Reply:
will it be the same in the UI and the database? Thats the most important thing. If the database clock and frontend timezones are different and we dont handle it, it can cause a lot of issues with these expiring things!